docs/application/web/guides/connectivity/bluetooth.md

```diff
@@ -697,8 +757,531 @@ To read and write a value of the Bluetooth descriptor:
    descriptor.writeValue(newValue, writeSuccess, writeFail);
    ```
 
+### Accessing the ATT MTU of connected device
```

Comment:
Suggestion:

### Access ATT MTU of connected device

Reply:
I have applied your suggestions.